project/src/main/java/com/google/sps/servlets/SignUpServlet.java

```diff
@@ -72,7 +72,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         userEntity.setProperty("school", school);
         userEntity.setProperty("phone", phone);
         userEntity.setProperty("metric", "fahrenheit");
-        userEntity.setProperty("admin", false);
+        userEntity.setProperty("admin", true);
```

Comment:
Probably change this back to false if you already made your test admin account!

Reply:
done!